packages/kbn-plugin-helpers/package.json

```diff
@@ -1,43 +1,32 @@
 {
   "name": "@kbn/plugin-helpers",
-  "version": "9.0.2",
+  "version": "1.0.0",
   "private": true,
   "description": "Just some helpers for kibana plugin devs.",
   "license": "Apache-2.0",
-  "main": "target/lib/index.js",
-  "scripts": {
-    "kbn:bootstrap": "tsc"
-  },
+  "main": "target/index.js",
   "bin": {
     "plugin-helpers": "bin/plugin-helpers.js"
   },
+  "scripts": {
+    "kbn:bootstrap": "rm -rf target && tsc",
+    "kbn:watch": "tsc --watch"
+  },
   "dependencies": {
-    "@babel/core": "^7.11.1",
-    "argv-split": "^2.0.1",
-    "commander": "^3.0.0",
+    "@kbn/dev-utils": "1.0.0",
+    "@kbn/optimizer": "1.0.0",
```

Comment:
Aren't dev-utils and optimizer devDependencies?

Reply:
Not really, since this package can't function without them.

Reply:
I think that case could be made for all dev dependencies. I think it's fine since `@kbn/plugin-helpers` itself is a dev dependency.

Reply:
If this package was published to npm it would require `@kbn/optimizer` and `@kbn/dev-utils` to function, they're not purely for development tooling while working on the package, which is how I interpret devDependencies